appengine-java11/http-server/pom.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?><project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>com.example.appengine</groupId>
+  <artifactId>http-server</artifactId>
+  <version>0.1</version>
+  <properties>
```

Comment:
Please add the parent POM (And run formatting on this file)

Reply:
done.